Makefile

```diff
@@ -54,3 +54,5 @@ clean: ## Clean build directories
 	rm -rf out
 	rm -rf tmp
 	./gradlew clean
+
+launch: clean build run
```

Comment:
Could you add some help string to this so it shows up when you run make help, e.g.

    launch: clean build run ## add some help text after two hash symbols